scripts/validateTimestamp.js

```diff
@@ -0,0 +1,38 @@
+const hre = require("hardhat");
```

Comment:
Duplicate code. Reuse the code.

Reply:
fixed https://github.com/razor-network/bridge-proxy/pull/70/commits/4f83d9be6a89aa62784f4603cbc69a3cb9fb9f50